.github/workflows/bump_plugins.yml

```diff
@@ -0,0 +1,107 @@
+name: Update plugins
+on:
+  workflow_dispatch:
+
+jobs:
+  # ugly way to share the plugin list
+  plugin_list:
+    runs-on: ubuntu-latest
+    outputs:
+      plugins: ${{ steps.set-list.outputs.plugins }}
+    steps:
+    - name: Write the plugin list
+      run: echo '["foreman_ansible"]' > plugin_list.json
```

Comment:
How will you deal with plugins which are only packaged on RPMs or (less common) only on deb? I think it may be needed to write two workflows: one for RPMs and one for deb.

Reply:
the plugin list gets feeded into two jobs, there are adjustments possible for each of them.

Reply:
My point is that the list of plugins packaged on one is different than the other. I don't think we'll see a completely matching list on both.

Another thing I didn't think about before: on RPMs you also have plugins in `packages/katello`. Just 3 now (since we don't bump Katello itself), but `rubygem-foreman_rh_cloud`, `rubygem-foreman_scc_manager` and `rubygem-foreman_virt_who_configure` do live there because they depend on Katello.

Reply:
the difference between DEB and RPM can be addressed easily, this would be just the common ones, and for each job respectively we would add additional entries.

The katello packages would be a bit of an issue, but it just adds a bit of complexity to determinig the directory, right?

Reply:
It could exit early if it isn't found, that would be one solution, but you'll probably find this issue quickly.

For Katello you can probably work around it by calling `bump_rpm.sh` with `packages/*/$package` and let shell globbing solve it.